src/NServiceBus.Testing.Fakes/TestableRoutingContext.cs

```diff
@@ -20,5 +21,10 @@ public partial class TestableRoutingContext : TestableBehaviorContext, IRoutingC
         /// The routing strategies for the operation to be dispatched.
         /// </summary>
         public IReadOnlyCollection<RoutingStrategy> RoutingStrategies { get; set; } = new RoutingStrategy[0];
+
+        /// <summary>
+        /// A <see cref="T:NServiceBus.Extensibility.ContextBag" /> which can be used to extend the current object.
+        /// </summary>
+        public new ContextBag Extensions { get; set; } = new ContextBag();
```

Comment:
Isn't the property already available https://github.com/Particular/NServiceBus/blob/13ecd6c5c36752d01b64dc2eb895e13ca189a207/src/NServiceBus.Testing.Fakes/TestableBehaviorContext.cs#L19 ?